params/governance_params.go

```diff
@@ -90,6 +95,11 @@ var (
 	DefaultProposerPolicy          = uint64(RoundRobin)
 	DefaultSubGroupSize            = uint64(21)
 	DefaultUnitPrice               = uint64(250000000000)
+	DefaultLowerBoundBaseFee       = uint64(25000000000)
+	DefaultUpperBoundBaseFee       = uint64(750000000000)
+	DefaultGasTarget               = uint64(30000000)
+	DefaultBlockGasLimit           = uint64(84000000)
+	DefaultBaseFeeDenominator      = uint64(36)
```

Comment:
Please describe how these values were derived into the mainboard of the PR.

Reply:
You can find KIP-71 history in https://github.com/klaytn/kips/pull/72

Reply:
The proposal seems to lack grounds for those initial values. Anyway, thank you.

Reply:
@hyunsooda  If you want to know why we derive the initial values like that in detail, please invite me for meeting.
It is derived from some historical back data and a little bit heuristic experience in klaytn.

Reply:
Is there a public link for that history? If so, please put the link into the description of the PR so that anyone can figure it out and make sure to understand it.

Reply:
I'm so sorry there are no documents to show you summarized history. I think we can treat this issue by meeting.

Reply:
Okay. Next time, please describe these surprising changes such as these static values, because the new readers to your change are unaware of the behind the history of this decision. If the code is not obvious itself, please describe them in the description of the PR.